ImputationPipeline/ScoringPart.wdl

```diff
@@ -304,47 +305,124 @@ task AdjustScores {
 
 			population_data = merge(population_pcs, population_scores, by.x="IID", by.y="X.IID")
 
-			# generate the linear model from the population data using the first 5 PCs
-			population_model = glm(SCORE1_SUM ~ PC1 + PC2 + PC3 + PC4 + PC5, data = population_data, family = "gaussian")
+			# generate the linear model from the population data using the first 4 PCs
+			population_model = glm(SCORE1_SUM ~ PC1 + PC2 + PC3 + PC4, data = population_data, family = "gaussian")
```

Comment:
On my future wishlist is not hardcoding number of pcs, but definitely not priority

Reply:
👍